R/render_docs.R

```diff
@@ -46,7 +47,14 @@
 #'   render_docs(parallel = TRUE)
 #'
 #' }
-render_docs <- function(path = ".", verbose = FALSE, parallel = FALSE, freeze = FALSE, autolink = FALSE) {
+render_docs <- function(path = ".", verbose = FALSE, parallel = FALSE, freeze = FALSE, autolink = FALSE, ...) {
+
+  # Quarto sometimes raises errors encouraging users to set `quiet=FALSE` to get more information. 
+  # This is a convenience check to match Quarto's `quiet` and `altdoc`'s `verbose` arguments.
+  dots <- list(...)
+  if ("quiet" %in% names(dots) && is.logical(dots[["quiet"]]) && length(dots[["quiet"]]) == 1) {
```

Comment:
I think the length condition should be before the `is.logical()` to ensure we won't have an issue when `is.logical()` returns a vector of length > 1

Reply:
Good catch. But I think the proper fix is to have the `length` check in `isTRUE()`, because `length()` might not work at all or weirdly on objects of certain class.